apps/mochi-web/hooks/app/useFetchProfileTransaction.ts

```diff
@@ -0,0 +1,22 @@
+import useSWR from 'swr'
+import { API, GET_PATHS } from '~constants/api'
+import { ViewTransferV2Response } from '~types/mochi-pay-schema'
+
+export const SWR_PROFILE_TRANSACTION_KEY = 'SWR_PROFILE_TRANSACTION_KEY'
+
+export const useFetchProfileTransaction = (
+  id: string,
+  allowFetch: boolean = true,
+) => {
```

Comment:
Do we need add filters and pagination query to api? I see you filter in FE and do not have pagination. Pls confirm @zlatanpham